documentation/src/main/asciidoc/reference/indexing-massindexer.asciidoc

```diff
@@ -333,6 +333,14 @@ or for more advanced use cases, such as cancelling mass indexing on the first fa
 
 Implementations of `MassIndexingFailureHandler` must be thread-safe.
 
+|`massIndexingEnvironment(MassIndexingEnvironment)`
```

Comment:
If we want to be consistent with other pluggable components (monitor, failure handler) we should not use the prefix for this method, just for the interface.

```suggestion
|`environment(MassIndexingEnvironment)`
```

Reply:
yeah, I had that intent initially, but `environment` seems to be too generic and sounds "too powerful", but if it's just me then I'm happy to make a change 😄